packages/pluggableWidgets/video-player-native/src/VideoPlayer.tsx

```diff
@@ -36,27 +53,129 @@ export function VideoPlayer(props: VideoPlayerProps<VideoStyle>): ReactElement {
         setStyles(alteredStyles);
     }, [props.style, props.aspectRatio, videoAspectRatio]);
 
+    useEffect(() => {
+        if (props.showControls) {
+            showControlsHandler();
+        }
+    }, [fullScreen, props.showControls]);
+
+    function showControlsHandler(): void {
```

Comment:
Using `useCallback` here will optimize the effect [above](https://github.com/mendix/widgets-resources/pull/1303/files#diff-5b19cf908530b18852147d085549427ea128e2eb1da3dddad52826adb3d6aae7R56)

Reply:
done